### DISCUSSION TITLE
bagaimana caranya handle kata umum yang disingkat dalam konteks sentimen analysis

### DISCUSSION THREAD

**Discussion by Gustana** [teamalgoritma/community#694] [Q&A] [answered]
misalnya ada komentar: sy ska skli produk ini.

bagaimana cara handle komentar ini supaya bentuknya menjadi, saya suka sekali produk ini?

**Comment by finesaaa** [comment, marked as answer]
Dear @Gustana,
 Untuk mengubah komentar "sy ska skli produk ini" menjadi "saya suka sekali produk ini", Anda dapat melakukan normalisasi dengan menggunakan kamus-kamus yang tersedia di internet *mengganti singkatan menjadi kata baku", saya akan memberikan beberapa referensi ya pak:

-  https://github.com/okkyibrohim/id-abusive-language-detection/blob/master/kamusalay.csv
- Ini dokumen saya punya, saya lupa referensinya dari mana :) [formal.csv](https://github.com/teamalgoritma/community/files/11786898/formal.csv)

Semoga membantu!

**Reply by Gustana** [reply]
baik, terimakasih kak
